src/test/java/tool/designpatterns/verifiers/multiclassverifiers/AdapterVerifierTest.java

```diff
@@ -0,0 +1,100 @@
+package tool.designpatterns.verifiers.multiclassverifiers;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+import static org.gradle.internal.impldep.org.junit.Assert.assertFalse;
+import static org.gradle.internal.impldep.org.junit.Assert.assertTrue;
+
+import com.github.javaparser.ast.NodeList;
+import com.github.javaparser.ast.body.ClassOrInterfaceDeclaration;
+import com.github.javaparser.ast.type.ClassOrInterfaceType;
+
+import org.junit.jupiter.api.Test;
+import tool.designpatterns.Pattern;
+import utilities.TestHelper;
+
+public class AdapterVerifierTest {
+
+    @Test
+    public void testVerifyImplementsAdapter() throws IOException {
+        ClassOrInterfaceDeclaration adapter = TestHelper.getMockCompUnit(
+            "adapter" + "/implementsAdapter", "MicroUsbToLightningAdapter").findAll(
+            ClassOrInterfaceDeclaration.class).get(0);
+        ClassOrInterfaceDeclaration adaptee = TestHelper.getMockCompUnit(
+            "adapter/implementsAdapter", "LightningPhone").findAll(
+            ClassOrInterfaceDeclaration.class).get(0);
+        HashMap<Pattern, List<ClassOrInterfaceDeclaration>> patternGroup = createPatternGroup(
+            adapter, adaptee);
+
+        System.out.println(new AdapterVerifier().verifyGroup(patternGroup).getFullMessage());
```

Comment:
It is best practice to avoid using Sout in test etc. If it is relevant a static Logger should be created which can be integrated with gradle.

Reply:
It was not supposed to be there for the pr... fixed